testcrate/src/bench.rs

```diff
@@ -17,9 +17,10 @@ pub fn skip_sys_checks(test_name: &str) -> bool {
         "extend_f16_f32",
         "trunc_f32_f16",
         "trunc_f64_f16",
-        // FIXME(f16_f128): rounding error
+        // FIXME(#616): re-enable once fix is in nightly
         // <https://github.com/rust-lang/compiler-builtins/issues/616>
-        "mul_f128",
+        "mul_f32",
+        "mul_f64",
```

Comment:
Why will these be fixed with the nightly bump? I guess if `__mul[sd]f3` don't exist since they are expected to lower to assembly, then linking the `extern "C"` symbols is linking to the weak ones from this library?